lib/neo4_ecto.ex

```diff
@@ -1,27 +1,78 @@
 defmodule Neo4Ecto do
   @moduledoc """
-  Neo4j adapter for Ecto.
+  Neo4Ecto is a Neo4j Adapter for Ecto.
 
-  Handle Ecto behaviours implementations in a way that enables us
-  to use Neo4j like we use Ecto daily.
+  Through Neo4Ecto we can do normal Ecto implementations to connect
+  and manage our Neo4j database in a much simpler way.
 
-  Currently, this implementation covers:
-  @Ecto.Adapter -> So you can use Neo4Ecto as your repo adapter
+  Currently Neo4Ecto handles 3 of the main components from Ecto:
+    * `Ecto.Adapter`
+    * `Ecto.Adapter.Schema`
+    * `Ecto.Adapter.Storage`
 
-  ## Example
+  ## Config Example
 
-        defmodule MyApp.Repo do
-          use Ecto.Repo, otp_app: :my_app, adapter: Neo4Ecto
+  Set `Neo4Ecto` as the default Ecto Adapter inside your Repo module:
+
+      defmodule MyApp.Repo do
+        use Ecto.Repo, otp_app: :my_app, adapter: Neo4Ecto
+      end
+
+  Add the proper configuration for Ecto to recognize your Neo4j
+  Database, you may want something like this in your `config/config.exs`:
+
+      config :my_app, ecto_repos: [MyApp.Repo]
+
+      config :my_app, Repo,
+        hostname: "localhost",
+        basic_auth: [username: "neo4j", password: "password"]
```

Comment:
Just to keep the same pattern as described on README.md
```suggestion
        basic_auth: [username: "neo4j", password: "password"],
        pool_size: 5
```